fixtures/allauth.json

```diff
@@ -0,0 +1,32 @@
+[
```

Comment:
Now I am thinking this might break when you git push and Digital Ocean App tries to use it? I'm not sure. So please test the Digital Ocean App deploy.

Reply:
I will check this on digital ocean

Reply:
I have tested this on digital ocean app and it works good